Add 'nameSuffix' parameter for configuring generated service-class names (#1966)
square/wire: wire-library/docs/wire_compiler.md

    // `none` to not generate services.
    rpcRole = 'server'

>     // When null, generated service types will have a suffix appended to
>     // their names, inferred from rpcRole - 'Server' for 'server',
>     // 'Client' for 'client', etc. When non-null, generated service types
>     //  will have the given value for a suffix.
>     nameSuffix = null

Reviewer: The list has only non-default values so let's use one.

```suggestion
    // If set, the value will be appended to generated service type names. If null, their rpcRole
    // will be used as a suffix instead.
    nameSuffix = "Suffix"
```
